tests/Test/Runner/Tests.elm

```diff
@@ -0,0 +1,200 @@
+                                                \() -> Expect.fail "failed on purpose"
+                                            , Test.skip <|
+                                                test "is skipped" <|
+                                                    \() -> Expect.fail "failed on purpose"
+                                            ]
+                                    ]
+                    in
+                        case runners of
+                            Skipping runners ->
+                                runners
+                                    |> List.length
+                                    |> Expect.equal 1
+
+                            val ->
+                                Expect.fail ("Expected SeededRunner to be Skipping, but was " ++ toString val)
+            , test "a pair of tests where one uses skip is a Skipping summary" <|
+                \() ->
+                    let
+                        runners =
+                            toSeededRunners <|
+                                describe "two tests"
+                                    [ test "passes" <|
+                                        \() -> Expect.pass
+                                    , Test.skip <|
+                                        test "fails" <|
+                                            \() -> Expect.fail "failed on purpose"
+                                    ]
+                    in
+                        case runners of
+                            Skipping runners ->
+                                runners
+                                    |> List.length
+                                    |> Expect.equal 1
+
+                            val ->
+                                Expect.fail ("Expected SeededRunner to be Skipping, but was " ++ toString val)
+            , test "when all tests are skipped, we get an empty Skipping summary" <|
+                \() ->
+                    case toSeededRunners (Test.skip <| test "passes" (\() -> Expect.pass)) of
+                        Skipping runners ->
+                            runners
+                                |> List.length
+                                |> Expect.equal 0
+
+                        val ->
+                            Expect.fail ("Expected SeededRunner to be Skipping, but was " ++ toString val)
+            , test "a test that does not use only or skip is a Plain summary" <|
+                \() ->
+                    case toSeededRunners (test "passes" (\() -> Expect.pass)) of
+                        Plain runners ->
+                            runners
+                                |> List.length
+                                |> Expect.equal 1
+
+                        val ->
+                            Expect.fail ("Expected SeededRunner to be Plain, but was " ++ toString val)
+            ]
+        ]
+
+
+passing : Test
+passing =
+    test "A passing test" (\() -> Expect.pass)
```

Comment:
Done, but renamed it to `expectPass` (I thought `passingTest` was a weird thing to call a value of type `Expectation`)